<launch>
  <node pkg="bandit_ik" type="bandit_arm_kinematics_node" name="bandit_left_arm_kinematics" output="screen">    
   <param name="tip_name" value="left_hand_link" />     
   <param name="root_name" value="bandit_torso_link" />  
  </node>
</launch>
